test/controllers/medias_controller_test.rb

```diff
@@ -376,11 +332,6 @@ def setup
     end
   end
 
-  test "should not clear cache if not authenticated" do
-    delete :delete, url: 'http://test.com', format: 'json'
-    assert_response 401
-  end
-
```

Comment:
@daniel-shuy why did you remove these tests? They were not affected by your changes, they are also passing.
Could you add them back, please?